ta/pkcs11/src/persistent_token.c

```diff
@@ -538,6 +539,40 @@ void release_persistent_object_attributes(struct pkcs11_object *obj)
 	obj->attributes = NULL;
 }
 
+enum pkcs11_rc update_persistent_object_attributes(struct pkcs11_object *obj)
+{
+	TEE_Result res = TEE_ERROR_GENERIC;
+	TEE_ObjectHandle hdl = TEE_HANDLE_NULL;
+	uint32_t tee_obj_flags = TEE_DATA_FLAG_ACCESS_WRITE;
+	size_t size = 0;
+
+	assert(obj && obj->attributes);
+
+	res = TEE_OpenPersistentObject(TEE_STORAGE_PRIVATE,
+				       obj->uuid, sizeof(*obj->uuid),
+				       tee_obj_flags, &hdl);
+	if (res) {
+		EMSG("OpenPersistent failed %#"PRIx32, res);
+		return tee2pkcs_error(res);
+	}
+
+	res = TEE_SeekObjectData(hdl, 0, TEE_DATA_SEEK_SET);
```

Comment:
In the specification for TEE_OpenPersistentObject:
`The initial data position in the data stream is set to 0.`

So we don't need to do the seek to start.

Reply:
Thanks, you are right. I will remove the Seek call. @etienne-lms , is that ok with you also ?

Reply:
yes, agree.